lib/recaptcha/token.rb

```diff
@@ -0,0 +1,19 @@
+module Recaptcha
+  module Token
+    require 'json'
+
+    def self.secure_token
+      private_key  =  Recaptcha.configuration.private_key
+      raise RecaptchaError, "No private key specified." unless private_key
+
+      stoken_json = {'session_id' => SecureRandom.uuid, 'ts_ms' => (Time.now.to_f * 1000).to_i}.to_json
```

Comment:
used to be 1 big ball and that ball was tested, now it's a new public api that people will use independently, so I'd prefer to have at least a basic smoke test for it ... not really necessary, just out of habit / best practice